src/GitLabApiClient/Models/AccessLevel.cs

```diff
@@ -0,0 +1,13 @@
+using System;
+
+namespace GitLabApiClient.Models
+{
+    public static class AccessLevel
```

Comment:
can you change this to an enum so the library users can find AccessLevel easier?

Reply:
separated classes